### DISCUSSION TITLE
Code generation for 3rd-party plugins

### DISCUSSION THREAD

**Discussion by BernhardGlueck** [nxrighthere/UnrealCLR#161] [General]
I have recently switched to UE5 EA 2.
In conjunction i am using the excellent VoxelPlugin third party plugin.

Now i am wondering, is there a way to generate wrappers for the VoxelPlugin APIS ( UObject based ) 
for UnrealCLR ? 

I tried setting up a project with voxelplugin ( plugin is built, and useable from the editor and runtime ) 
and then installing UnrealCLR, but the resulting Unreal.Framework only contains classes for Core UE ... 

Am i missing something or is that scenario just not supported ?

**Comment by nxrighthere** [comment]
Support of 3rd-party plugins is out of scope of the project at the moment (see https://github.com/nxrighthere/UnrealCLR/discussions/56#discussioncomment-417105).
